src/System.Net.WebSockets.Client/tests/testloop.bat

```diff
@@ -0,0 +1,14 @@
+SETLOCAL EnableDelayedExpansion
+SETLOCAL ENABLEEXTENSIONS
+
+for /L %%x in (1,1,500) do (
+
+mkdir "D:/TestDiag/Run %%x"
```

Comment:
I don't recommend we hard code drive letters in these scripts.  Not everyone has a "D:" drive.  I don't for example.  I have my other drives set to letters like "S:" (enlistments) and "W:" (work stuff)